lsp-mode.el

```diff
@@ -5498,7 +5498,8 @@ textDocument/didOpen for the new file."
 PARAMS are the `workspace/configuration' request params"
   (->> items
        (-map (-lambda ((&ConfigurationItem :section?))
-               (gethash section? (lsp-configuration-section section?))))
+               (apply 'ht-get* (append (list (lsp-configuration-section section?))
```

Comment:
This can be 
```suggestion
               (apply 'ht-get* (nconc (list (lsp-configuration-section section?))
```
since you've already create a new list for the first item. `append` create new list.

Or you can write that like `` `(,lsp-configuration-section section) ,@(split-string section? "\\."))`` which doesn't require writing `append` or `nconc`